src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -189,7 +189,7 @@
  * In the JDK Reference Implementation, the virtual thread scheduler may be configured
  * with the following system properties:
  * <table class="striped">
- * <caption style="display:none:">System properties</caption>
+ * <caption style="display:none;">System properties</caption>
```

Comment:
Just out of curiosity, is that semi-colon even needed, or would `display:none` be enough?

Reply:
Now that I think about it, do you know why the `display:none` is being set here? I have very limited knowledge of CSS, wouldn't the use of `display:none` make the `System Properties` caption text invisible?

Reply:
Indeed, the semi-colon is optional at the end of an inline CSS attribute. I removed it in my follow-up commit. 

Regarding the presence of invisible `<caption>` elements in HTML tables: this is indeed questionable. I assume it was based on the notion that a `<caption>` element is required in HTML tables, which is not the case (neither for HTML4 nor HTML5). However, it is quite pervasive (there are around 170 occurrences of `<caption style="display:none">` in OpenJDK source), and one could argue that it serves as an informational comment both in Java and HTML.